src/Nox.Lib/Factories/Types/NoxTypeColorFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Nox.Solution;
+using Nox.Types;
+
+namespace Nox.Factories.Types
+{
+    public class NoxTypeColorFactory : NoxTypeFactoryBase<Color>
+    {
+        public NoxTypeColorFactory(NoxSolution solution) : base(solution)
+        {
+        }
+
+        public override Color? CreateNoxType(NoxSimpleTypeDefinition simpleTypeDefinition, dynamic? value)
```

Comment:
I think the dynamic value would always come in whatever type we are persisting in the Database, for instance in this case Color is persisted as an array of bytes, so I believe we should only instantiate the Color.From(byte[]).
@rochar will need your confirmation here please.

Reply:
![image](https://github.com/NoxOrg/Nox.Generator/assets/131167535/ad767cd6-a8d9-4b51-b528-0065d3099067)

Reply:
Factory and From method updated.